.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+php:
+  - 7.4
```

Comment:
Is good put try another PHP versions and put the minimum stability in composer.json, like [this](https://github.com/LyseonTech/dou-collector/blob/master/composer.json#L8) line.

I suggest to see the integration with TravisCI in your TravisCI. Go to https://travis-ci.com/, loggin with your github account, enable the forked repository and see the result of tests

Reply:
I didn't know about this. Since I run your project with Docker that uses php:7.4-fpm, I was thinking that only php7.4 was necessary. But I put the php7.3 too and all tests pass in Travis.

Reply:
perfect